packages/mdc-textfield/label/foundation.js

```diff
@@ -58,32 +58,32 @@ class MDCTextFieldLabelFoundation extends MDCFoundation {
     return this.adapter_.getWidth();
   }
 
-  /** Makes the label float above the text field. */
-  floatAbove() {
-    this.adapter_.addClass(cssClasses.LABEL_FLOAT_ABOVE);
-    this.adapter_.removeClass(cssClasses.LABEL_SHAKE);
-  }
-
   /**
-   * Deactivates the label's focus state based on whether the text
-   * field input is empty and passes validity checks.
-   * @param {boolean} shouldRemoveLabelFloat
+   * Styles the label to match the supplied optional params. If a
+   * param is not supplied, the method makes no changes related to that param.
+   * @param {string=} optValue
+   * @param {boolean=} optIsValid
+   * @param {boolean=} optIsBadInput
+   * @param {boolean=} optIsFocused
    */
-  deactivateFocus(shouldRemoveLabelFloat) {
-    this.adapter_.removeClass(cssClasses.LABEL_SHAKE);
+  style(optValue, optIsValid, optIsBadInput, optIsFocused) {
```

Comment:
Document this in the mdc-textfield/label/README.md

...actually see my above comment first, about maybe making this more clear...but then we should document all new public methods in mdc-textfield/label/README.md

Reply:
Done.